app/views/user/_tracks_progress.erb

```diff
@@ -0,0 +1,22 @@
+<% if user_tracks_summary.any? %>
+  <h3> Progess </h3>
+  <% if params[:all] == 'true' %>
+    <% user_tracks_summary.each_slice(3) do |batch| %>
+      <div class="row bottom-margin">
+        <% batch.each do |track_info| %>
+          <%= erb :"user/_track_progress", locals: { track_info: track_info } %>
+        <% end %>
+      </div>
+    <% end %>
+    <a href="?all=false"> Fewer Languages </a>
+  <% else %>
+    <div class="row bottom-margin">
+      <% user_tracks_summary.each_slice(3).first.each do |track_info| %>
+        <%= erb :"user/_track_progress", locals: { track_info: track_info } %>
+      <% end %>
+    </div>
+    <% if user_tracks_summary.size > 1 %>
```

Comment:
What happens if the user has 2 tracks, this link should exist? Sorry if I am misinterpreting this.